apps/foosball-api/src/app/app.module.ts

```diff
@@ -9,9 +9,21 @@ import { PlayersController } from './controllers/players.controller';
 import { AuthController } from './controllers/auth.controller';
 import { DataController } from './controllers/data.controller';
 import { MatchResultsController } from './controllers/match-results.controller';
+import { FirebaseNestModule } from '@foosball/firebase-nest';
 
 @Module({
-  imports: [DataModule, CoreModule, AuthModule],
+  imports: [
+    DataModule,
+    CoreModule,
+    AuthModule,
+    FirebaseNestModule.forRoot({
```

Comment:
On a next iteration, we can leverage the ConfigService (NestJS)

`import { ConfigService } from '@nestjs/config';`

As an example, see `main.ts` in this app.